theme/static/js/hs-vue/left-header-app.js

```diff
@@ -122,6 +122,12 @@ Vue.component('add-author-modal', {
             },
         }
     },
+    mounted(){
+        let that = this;
```

Comment:
@devincowan Notice what we did here. The element inside of the selector is the parent that is expected to be rendered when the Vue instance is mounting. After that the parameter `.remove` is the selector for an element this is dynamically generated.

Reply:
@Maurier that makes perfect sense, appreciate the explanation